addOns/reports/src/main/java/org/zaproxy/addon/reports/sarif/SarifProtocolData.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.reports.sarif;
+
+/**
+ * Internal class to hold protocol and version separated. Provides also parse functionality inside.
+ */
+class SarifProtocolData {
+
+    private String protocol;
+    private String version;
+
+    private SarifProtocolData() {
+        // private constructor - so static method must be used
+    }
```

Comment:
Per the comment shouldn't the variables and methods be static?

Reply:
I will change the comment to
```
 // private constructor - so static parse method must be used to create the protocol data object
```
so it is more clear: You have different objects, so getters may not be static, just the creation is done in a static method, so by the name of the method it is clear how the protocol data is being constructed.

Reply:
Ok

Reply:
Or just remove the comment, that the constructor is private already indicates that other method needs to be used.